drivers/bridge/setup_device.go

```diff
@@ -25,7 +27,10 @@ func setupDevice(config *networkConfiguration, i *bridgeInterface) error {
 	// Only set the bridge's MAC address if the kernel version is > 3.3, as it
 	// was not supported before that.
 	kv, err := kernel.GetKernelVersion()
-	if err == nil && (kv.Kernel >= 3 && kv.Major >= 3) {
+	if err != nil {
+		return fmt.Errorf("failed to check kernel versions: %v", err)
+	}
```

Comment:
Is there a reason to fail the operation if GetKernelVersion fails now while it was a bit lenient (and rightly so) in the existing code ?